Sprint-1/errors/0.js

```diff
@@ -1,2 +1,5 @@
 This is just an instruction for the first activity - but it is just for human consumption
-We don't want the computer to run these 2 lines - how can we solve this problem?
\ No newline at end of file
+We don't want the computer to run these 2 lines - how can we solve this problem?
+console.log()
+//SyntaxError: Unexpected identifier
+//It misses certain of java script structure like keywords ,variables and functions
```

Comment:
So how can we keep these two lines but prevent them from being "seen" by the JavaScript interpreter?